src/testing-utils.ts

```diff
@@ -49,6 +49,12 @@ export function setupTests(test: TestInterface<any>) {
     const processStderrWrite = process.stderr.write.bind(process.stderr);
     t.context.stderrWrite = processStderrWrite;
     process.stderr.write = wrapOutput(t.context) as any;
+
+    // Many tests modify environment variables. Take a copy now so that
+    // We reset them after the test to keep tests independent of each other.
+    // process.env only has strings fields, so a shallow copy is fine.
+    t.context.env = {};
+    Object.assign(process.env, t.context.env);
```

Comment:
```suggestion
    Object.assign(t.context.env, process.env);
```

Is this the wrong way round?  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign says:

> The Object.assign() method copies all enumerable own properties from one or more source objects to a target object. It returns the target object.
> `const returnedTarget = Object.assign(target, source);`